release_description.md

```diff
@@ -0,0 +1,14 @@
+The Amazon QLDB team is pleased to announce the release of v2.0.0 of the Go QLDB driver.
```

Comment:
I don't see a GitHub CLI command to update a release (since you released it yesterday). I would forgo following the MCM instructions (since it is only for creating new releases...) and just manually edit the release description of v2.0.0 release

Reply:
Removed file.